src/routes/(console)/project-[project]/functions/function-[function]/settings/updateRuntime.svelte

```diff
@@ -51,11 +67,12 @@
                 $func.providerRepositoryId || undefined,
                 $func.providerBranch || undefined,
                 $func.providerSilentMode || undefined,
-                $func.providerRootDirectory || undefined
+                $func.providerRootDirectory || undefined,
+                specification
             );
             await invalidate(Dependencies.FUNCTION);
             addNotification({
-                message: 'Runtime has been updated',
+                message: 'Runtime settings have been updated',
```

Comment:
Below this, in HTML, line 93, it says `Runtime` as title of the card. We might want to rename this to "Settings" too, possibly.

Also important to understand how specs behave during build. Is gb-hours increased properly? Is the limit used properly?

Best to have it explained in Console, or docs.